### ISSUE TITLE
Rectification factor in location decoder

### ISSUE THREAD

**Issue by ericzzj1989** [wangch-g/lanet#2]
Thanks for your great work.

The location decoder has a learnable rectification factor you mentioned in the original paper, however, this learning parameter is not reflected in the code.
https://github.com/wangch-g/lanet/blob/74dd27cafa2d3d76ba9ddcb3a8804728f9998653/network_v0/modules.py#L58
        self.loc_out = nn.Conv2d(256, 2, kernel_size=3, stride=1, padding=1)
The 'loc_out' just has 2 channels in the output as above.

Can you give me some advice?

**Comment by wangch-g** [comment]
https://github.com/wangch-g/lanet/blob/74dd27cafa2d3d76ba9ddcb3a8804728f9998653/network_v0/modules.py#L65
# cross_head:
self.shift_out = nn.Conv2d(256, 1, kernel_size=3, stride=1, padding=1)
The 'shift_out' is the rectification factor.

**Comment by wangch-g** [comment]
https://github.com/wangch-g/lanet/blob/74dd27cafa2d3d76ba9ddcb3a8804728f9998653/network_v0/modules.py#L65
self.shift_out = nn.Conv2d(256, 1, kernel_size=3, stride=1, padding=1)
The 'shift_out' is the rectification factor.

**Comment by ericzzj1989** [comment]
Got it. Many thanks for your reply.
